bf/bf.c

```diff
@@ -29,48 +27,78 @@ int main(int argc, char const *argv[])
 	else
 	{
 		FILE *bf_file = fopen(argv[1], "r");
-		FILE *c_output = fopen(argv[2], "w");
+        	int matches = 0;
```

Comment:
Also, `matches` might be a little ambiguous. Maybe `unmatched_braces` or `brace_stack` or something?